package.json

```diff
@@ -27,6 +27,7 @@
   },
   "homepage": "https://github.com/boleto-br/boleto-pdf#readme",
   "dependencies": {
+    "gulp": "^4.0.2",
```

Comment:
Could you remove this dependency? we already use [rollup ](https://rollupjs.org/guide/en/)for automate deploy.

Reply:
> @jansilva you still need resolve the conflicting files.

It seems that Iḿ not allowed to resolve conflict because de button is disabled for me. One of the files was moved from bradesco folder to commons(a new folder). The file is format-date.js. Iḿ a little confused..

Reply:
I will try to solve it locally and commit againt in branch..

Reply:
This is exactly what you need to do, resolve the conflicts locally and commit and push the changes.